buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -46,6 +46,7 @@
       "alwaysRun": false,
       "displayName": "Extract symbol packages; if release branch, archive",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), eq(variables['PublishToMyGet'], 'True'))",
```

Comment:
I think that the only reason this step is an inline powershell step is because Conditional's were not supported.  From a debugging / reading / simplicity perspective, it'd be much nicer if this was just the `build-managed.cmd -- blah` part and utilized the conditional.

Reply:
I agree it's not pretty, but the string methods exposed to Conditional Tasks don't seem to include something like startsWith, so unless you know that's defined I'd rather not guess at this and be wrong.